docs/changes/coordinates/13162.api.rst

```diff
@@ -0,0 +1 @@
+The dms_to_degrees function (and implicitly tuple-based initialization of ``Angle``) is now deprecated.
```

Comment:
Let's add a sentence of *why* it is deprecated. Something like ", as it was difficult to be sure about the intent of the user for signed values of the degrees, minutes, and seconds."

Reply:
Done!